src/other/Table.js

```diff
@@ -669,48 +695,47 @@
         }
     };
 
-    Table.prototype.applyHoverRowStyles = function(row){
-        var context = this;
-        row
-            .style("color",context.tbodyHoverRowFontColor())
-            .style("background-color",context.tbodyHoverRowBackgroundColor())
-        ;
-    };
-    Table.prototype.applySelectedRowStyles = function(row){
+    Table.prototype.getColumnFormatting = function(cellData,colIdx){
         var context = this;
-        row
-            .style("color",context.tbodySelectedRowFontColor())
-            .style("background-color",context.tbodySelectedRowBackgroundColor())
-        ;
-    };
-    Table.prototype.applyRowStyles = function(row, isFirstCol){
-        row
-            .style("color", isFirstCol ? this.tbodyFirstColFontColor() : this.tbodyFontColor())
-            .style("background-color", isFirstCol ? this.tbodyFirstColBackgroundColor() : this.tableZebraColor_exists() && this.data().indexOf(row.datum()) % 2 ? this.tbodyRowBackgroundColor() : this.tableZebraColor())
-        ;
-    };
-    Table.prototype.applyFirstColRowStyles = function(rows){
-        this.applyStyleToRows(rows,true);
+        if(typeof(cellData) === "string"){
```

Comment:
more than likely this block right here will be replaced by calls to the new DB object